cmd/flux/bootstrap.go

```diff
@@ -173,7 +174,12 @@ func bootstrapValidate() error {
 func mapTeamSlice(s []string, defaultPermission string) map[string]string {
 	m := make(map[string]string, len(s))
 	for _, v := range s {
-		m[v] = defaultPermission
+		if s := strings.Split(v, ":"); len(s) == 2 {
+			m[s[0]] = s[1]
+		} else {
+			m[v] = defaultPermission
+		}
 	}
```

Comment:
```suggestion
		m[v] = defaultPermission
		if s := strings.Split(v, ":"); len(s) == 2 {
			m[v] = s[1]
		}
	}
```

Reply:
You did not include the `m[v]` suggestion within your new commit. Using the same value origin everywhere makes it both easier to refactor things in the future, and easier to identify what value is set.